bedrock/firefox/templates/firefox/browsers/windows-64-bit.html

```diff
@@ -69,18 +78,12 @@ <h2>{{ ftl('windows-64-bit-how-do-you-get-64') }}</h2>
 class='mzp-t-product-firefox mzp-t-firefox mzp-t-dark'
 ) %}
 <div class="download-firefox">
-  {{ download_firefox(dom_id='win64-bottom-download', download_location='secondary cta') }}
+  {{ firefox_download_desktop_button_windows(cta_copy=ftl('windows-64-bit-download-firefox', fallback='download-button-download-firefox'), id='win64-bottom-download', build='64', position='secondary cta') }}
 </div>
 {% endcall %}
 
 {% endblock %}
 
-{% block sticky_promo %}
-  {% with non_fx_only=True %}
-    {% include '/includes/sticky-promo.html' %}
-  {% endwith %}
-{% endblock %}
+{% block sticky_promo %}{% endblock %}
```

Comment:
**Suggestion:** Maybe remove this `sticky_promo` block if we don't have one on the page?